tests/unit/test_bisector.py

```diff
@@ -12,7 +12,13 @@
 from mozregression.errors import LauncherError, MozRegressionError
 
 
+def mock_abstractmethod():
+    raise NotImplementedError
+
```

Comment:
Hi @parkouss , so I found a way to allow us to test the ABC which is patch the abstractmethod when we first init it. But I need to avoid call it before assign it to __abstractmethod__, because it will throw an exception anyway. like:
> tests/unit/test_test_runner.py:162: in <module>
>    class TestCommandTestRunner(unittest.TestCase):
> tests/unit/test_test_runner.py:164: in TestCommandTestRunner
>    @patch.multiple(test_runner.TestRunner, __abstractmethod__=mock_abstractmethod())
> tests/unit/test_test_runner.py:14: in mock_abstractmethod
>    raise NotImplementedError
> E   NotImplementedError
 
Do you have any ideal about it?